### DISCUSSION TITLE
posts/meme444

### DISCUSSION THREAD

**Discussion by giscus** [noaione/blog.n4o.xyz#31] [Blog Comment]
# posts/meme444

Lebih baik atau tidak?

https://blog.n4o.xyz/posts/meme444

**Comment by peke767** [comment]
Halo gan, saya suka encode anime dengan x265 mau mencoba ke 4:4:4 karna biasanya 4:2:0, setelah saya baca statment "4:4:4 tidak dapat bekerja dengan baik di resolusi ~1080p apalagi 4K, terlalu sedikit advantage yang bisa diambil dari 4:4:4 kalo 1080p+, jadi encoder biasanya ninggalin jadi 4:2:0 10bit aja"
jadi untuk res 1080p dan 2160p lebih baik stay di 4:2:0 10bit aja ya?
